src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -67,16 +67,20 @@ public FeedbackSessionResponseStatus() {
     }
 
     /**
-     * Returns list of students who did not respond to the feedback session.
+     * Returns list of students who did not respond to the feedback session
+     * sorted by teamName > studentNamelist.
      */
     public List<String> getNoResponse() {
+        noResponse.sort(compareByTeamNameStudentName);
         return noResponse;
     }
 
     /**
-     * Returns list of students who responded to the feedback session.
+     * Returns list of students who responded to the feedback session
+     * sorted by teamName > studentNamelist.
      */
     public List<String> getStudentsWhoResponded() {
+        studentsWhoResponded.sort(compareByTeamNameStudentName);
```

Comment:
Similar to above - revert the changes here and add a new method `getStudentsWhoRespondedSorted`, then use that where we need the sorted list.